test/CMakeLists.txt

```diff
@@ -101,21 +101,21 @@ list(APPEND TEST_LIST
 
 # This test does not run in Fast DDS 2.0.x because signal handling for closing the tool was not
 # backported.
-if(fastrtps_VERSION VERSION_GREATER_EQUAL "2.1.0")
+if(fastdds_VERSION VERSION_GREATER_EQUAL "2.1.0")
```

Comment:
Discovery Server is not going to build from this point onward if Fast DDS 3 is not used. Consequently, I think all these version checks in this and other files can be safely removed. @EduPonz, what do you think?